editor/src/components/editor/store/editor-state.ts

```diff
@@ -1941,6 +1942,21 @@ export function insertElementAtPath(
     indexPosition,
   )
 }
+export function insertElementsAtPath(
```

Comment:
Isn't this redundant? I only checked against my local code, but the type definitions of the two functions match.

Reply:
I think master only has a singular `insertElementAtPath`, which accepts a single `JSXElementChild`, while this version expects an array of `JSXElementChild`(ren) (but maybe I'm misunderstanding the problem)

Reply:
that's correct, this one receives an array of elements

Reply:
I meant in relation to the function it calls, `insertElementsAtPath` and `insertJSXElementChildren` have identical type signatures.

Reply:
okay I see the redundant problem now, just fixed it :)